### PR TITLE
Webui trigger actions

### PR DESCRIPTION
**Pull request by teosarca**

### CODE REVIEW

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/bpartner/interceptor/C_BPartner_Location.java [hunk 1]
```diff
@@ -74,9 +78,10 @@ public void afterChange(@NonNull final I_C_BPartner_Location bpLocation)
 			final I_C_Location oldLocation = oldLocationId != null ? locationDAO.getById(oldLocationId) : null;
 			final String oldRegionName = oldLocation != null ? oldLocation.getRegionName() : null;
 
-			// TODO: decide if we have an org change
-
-			Execution.getCurrent().requestFrontendToTriggerAction(moveToAnotherOrgTriggerAction(bpLocation));
+			if (!Check.isEmpty(oldRegionName) && !Check.isEmpty(newRegionName) && !oldRegionName.equals(newRegionName))
```
Comment (teosarca) [inline backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/bpartner/interceptor/C_BPartner_Location.java:81]:
```suggestion
					if (!Objects.equals(StringUtils.trimBlankToNull(oldRegionName), StringUtils.trimBlankToNull(newRegionName)))
```

**Review by siemiatj — APPROVED** [review]

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5585670_sys_gh10990_AskForOrgChange_SysConfig.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+-- 2021-04-15T12:56:01.539Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_SysConfig (AD_Client_ID,AD_Org_ID,AD_SysConfig_ID,ConfigurationLevel,Created,CreatedBy,EntityType,IsActive,Name,Updated,UpdatedBy,Value) 
+VALUES (0,0,541364,'S',TO_TIMESTAMP('2021-04-15 15:56:01','YYYY-MM-DD HH24:MI:SS'),100,'D','Y','AskForOrgChangeOnRegionChange',
```
Comment (metas-ts) [inline backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5585670_sys_gh10990_AskForOrgChange_SysConfig.sql:4]: please add a description, so one understand what happens if this is set to Y

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/controller/Execution.java [hunk 3]
```diff
@@ -221,6 +245,7 @@ private ExecutionBuilder()
 						}
 					}
 
+					assert versionException != null;
```
Comment (metas-ts) [inline backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/controller/Execution.java:248]: Asserts are pretty unusual in our code and i think they are not enabled when we run on the server.
To make is less surprising i suggest you use Check.assume

**Review by metas-ts — CHANGES_REQUESTED** [review]
@metas-rc

File: backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/controller/Execution.java (hunk 3 above)
Comment (metas-rc) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/controller/Execution.java:248]: @teosarca

Comment (teosarca) [reply backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/controller/Execution.java:248]: @metas-rc  pls change it
@metas-ts  let's discuss some time

File: backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5585670_sys_gh10990_AskForOrgChange_SysConfig.sql (hunk 2 above)
Comment (metas-rc) [reply backend/de.metas.adempiere.adempiere/migration/src/main/sql/postgresql/system/10-de.metas.adempiere/5585670_sys_gh10990_AskForOrgChange_SysConfig.sql:4]: Added

**Review by metas-ts — APPROVED** [review]

**Pull request merged**